docs/.nojekyll

```diff
@@ -0,0 +1 @@
+This file makes sure that Github Pages doesn't process mdBook's output.
```

Comment:
This file is not needed, it's sufficient to not enable github pages with jekyll in the repository setting and github pages will just serve the selected folder / branch

Reply:
It's automatically generated, and I'm not seeing a way to disable it.